build-wheelhouse/action.yml

```diff
@@ -92,6 +92,18 @@ runs:
       run: |
         python -m pip install ${{ env.install_target }}
 
+    - name: "Verify library is properly installed"
+      shell: bash
+      run: |
+        library_name=${{ inputs.library-name }}
+
+        if python -c "import $library_name"; then
+            echo "Python module '$library_name' successfully imported."
+        else
+            echo "Failed to import Python module '$library_name': Something went wrong."
+            exit 1
+        fi
```

Comment:
Not sure this would work... Your library name is ``ansys-mapdl-core`` but your import statement should be ``import ansys.mapdl.core as pymapdl`` for example

Reply:
veeeery good point.

Reply:
We use to have the other library name for that.... It's ok :)

Reply:
Fixed now in here: https://github.com/ansys/actions/pull/353/commits/b5b4a3eeab7f3c4c279b005a8618c4002b14560b


Code for local testing:

```bash
library_name="asdf-QERQER-asdf.asdfas"

library_name="${library_name//-/.}"
echo "Modified the library name for importing: '$library_name'"
```